content/docs/user-guide/basic-concepts.md

```diff
@@ -0,0 +1,34 @@
+# Basic Concepts of DVC
+
+DVC streamlines large data files and binary models into a single Git
+environment. This approach will not require storing binary files in your Git
+repository.
+
+![](/img/flow-large.png) _DVC data management_
+
+### Data Files
```

Comment:
what about models?

Reply:
This doc is very incomplete, leaving all this dor the PR that closes #72.

Reply:
yep, see my other comment ^^